mllib/src/main/scala/org/apache/spark/ml/regression/DecisionTreeRegressor.scala

```diff
@@ -220,17 +227,23 @@ class DecisionTreeRegressionModel private[ml] (
     var predictionColumns = Seq.empty[Column]
 
     if ($(predictionCol).nonEmpty) {
-      val predictUDF = udf { (features: Vector) => predict(features) }
+      val predictUDF = udf { vector: Vector => predict(vector) }
```

Comment:
Total nit but do you need to change these?

Reply:
to remove the unnecessary brackets, then I rename the var to keep in line with other places. I am neutral to revert these.